admin_manual/ai/overview.rst

```diff
@@ -175,6 +175,80 @@ Apps can integrate their content with Context Chat to make it available for quer
 * *files*
 * `Analytics <https://apps.nextcloud.com/apps/analytics>`_
 
+.. _ai-overview_improve-ai-task-pickup-speed:
+
+Improve AI task pickup speed
+----------------------------
+
+Most AI tasks will be run as part of the background job system in Nextcloud which only runs jobs every 5 minutes by default.
+To pick up scheduled jobs faster you can set up background job workers that process AI tasks as soon as they are scheduled.
+
+Screen or tmux session
+^^^^^^^^^^^^^^^^^^^^^^
+
+Run the following occ command inside a screen or a tmux session, preferably 4 or more times for parallel processing of multiple requests by different or the same user (and as a requirement for some apps like context_chat).
+It would be best to run one command per screen session or per tmux pane to keep the logs visible and the worker easily restartable.
```

Comment:
```suggestion
It would be best to run one command per screen session or per tmux window/pane to keep the logs visible and the worker easily restartable.
```